dashboard/app/views/home/_privacy.md

```diff
@@ -4,9 +4,9 @@
 <br />
 <br />
 
-Date of Last Revision: March 11, 2021
+Date of Last Revision: July 19, 2021
```

Comment:
Time traveler!

Reply:
Yeah, that's intentional - I'm going to merge it on Friday so that it makes it into the Monday DTP (July 19th)